### PR TITLE
ZIO2 - Akka 2.6

### PR DESCRIPTION
**Pull request by justcoon**
Akka 2.6.18

removed scala 2.11

### CODE REVIEW

**Comment by ghostdogpr** [comment]
There is a flaky test that is not new with your PR but prevents the CI from turning green, that is in ClusterSpec: `MemberUp(Member(akka://Test@127.0.0.1:2551, Up)) did not satisfy isSubtype(MemberLeft)`. Do you mind fixing the test?

**Comment by justcoon** [comment]
> There is a flaky test that is not new with your PR but prevents the CI from turning green, that is in ClusterSpec: `MemberUp(Member(akka://Test@127.0.0.1:2551, Up)) did not satisfy isSubtype(MemberLeft)`. Do you mind fixing the test?

hello, never seen this error locally, but it is look like that in this case, cluster event was not propagated to `queue`, at time, when `item` was taken from `queue`

so maybe some wait could solve it, 

or do you have some other idea how to solve this case?

**Comment by ghostdogpr** [comment]
Yeah, either wait or accept 1 or both messages.

**Comment by justcoon** [comment]
@ghostdogpr tests updated

**Comment by ghostdogpr** [comment]
Hmm, but now we're not checking that `MemberLeft` is emitted at all. How about making a `ZStream` from the queue and using `takeUntil` to take items until  we get `MemberLeft`?

**Comment by justcoon** [comment]
ok, changed it to `ZStream` as you proposed, with `takeUntil`
I tried `collectWhile` first, but build failed

File: src/test/scala/zio/akka/cluster/ClusterSpec.scala [hunk 1]
```diff
@@ -40,9 +43,15 @@ object ClusterSpec extends ZIOSpecDefault {
...
-            item  <- queue.take
-          } yield item
-        )(isSubtype[MemberLeft](anything)).provideLayer(ZLayer.fromManaged(actorSystem))
+            items <- ZStream
+                       .fromQueue(queue)
+                       .takeUntil {
+                         case _: MemberLeft => true
+                         case _             => false
+                       }
+                       .run(ZSink.collectAll)
```
Comment (ghostdogpr) [inline src/test/scala/zio/akka/cluster/ClusterSpec.scala]: minor: you can use `runCollect`

**Review by ghostdogpr — COMMENTED** [review]
Looks good! Just a minor comment

**Review by ghostdogpr — APPROVED** [review]
Thanks!

**Pull request merged**